Cargo.toml

```diff
@@ -22,20 +22,20 @@ default = []
 
 [dependencies]
 tower-service = "0.3"
-tower-load = "0.3"
+tower = { version = "0.4", features = ["load"] }
 futures-util = { version = "0.3", features = [ "sink" ] }
 futures-core = "0.3"
 futures-sink = "0.3"
-tokio = { version = "0.2", features = [ "sync" ] }
-crossbeam = "0.7"
+tokio = { version = "1.0", features = [ "sync" ] }
```

Comment:
I actually think we need the `rt` feature too to fix #18 (which is also a problem on `master`).